src/components/qas/questions/qaCalculator/qaCalculatorIcons.jsx

```diff
@@ -21,10 +21,9 @@ QACalculatorIcon.propTypes = {
 
 export const QACalculatorIconUnit = ({ unit, tiny, className }) => {
   return (
-    <span
-      className={classnames(icon, className, { [smallIcon]: !!tiny })}
-      dangerouslySetInnerHTML={renderDef(unit)}
-    />
+    <span className={classnames(icon, className, { [smallIcon]: !!tiny })}>
```

Comment:
If tiny is false by default I believe this can just be `classnames(icon, className, { [smallIcon]: tiny })` since even when not provided this prop would always be `true` or `false` right?

Reply:
solid catch here! Thank you!